ELLog/Logger.swift

```diff
@@ -60,11 +60,11 @@ public struct LogLevel: OptionSetType, BooleanType, CustomStringConvertible {
             options.append("VERBOSE")
         }
 
-        return options.joinWithSeparator(", ")
+        return options.joined(separator: ", ")
     }
 
     public init(rawValue: UInt) { self.rawValue = rawValue }
-    
+    //todo WARNING BooleanType has been removed
```

Comment:
BooleanType has been removed, I guess the function can still be there, there is just no protocol to conform to using in an if block is now more challenging. (No tests were broken here so..)